deploy/minikube/release_sanity_test.go

```diff
@@ -70,10 +72,43 @@ func TestReleasesJson(t *testing.T) {
 			actualSha, err := getShaFromURL(getDownloadURL(r.Name, platform))
 			if err != nil {
 				t.Errorf("Error calcuating SHA for %s-%s. Error: %s", r.Name, platform, err)
+				continue
 			}
 			if actualSha != sha {
 				t.Errorf("ERROR: SHA does not match for version %s, platform %s. Expected %s, got %s.", r.Name, platform, sha, actualSha)
+				continue
```

Comment:
Don't we want a bunch of errors if multiple platforms+shas fail?  I could see the first conditional needing a continue so that we don't check the SHA if we have trouble downloading it